bin/server/cmd/root.go

```diff
@@ -131,6 +141,11 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&flags.demoName, "demo-name", "", "name of the demo to use when running in demo mode")
 	rootCmd.PersistentFlags().BoolVar(&flags.debug, "debug", false, "whether to enable debug level logging")
 	rootCmd.PersistentFlags().StringVar(&flags.pgURL, "pg", "", "optional external PostgreSQL instance connection url(must provide dbname); for example postgresql://user:secret@masterhost:5432/dbname?sslrootcert=cert")
+
+	// Cloud backup related flags.
+	// TODO(dragonly): Add GCS usages when it's supported.
+	rootCmd.PersistentFlags().StringVar(&flags.backupBucket, "backup-bucket", "", "bucket where Bytebase stores backup data, e.g., s3:us-west-2//example-bucket. When provided, Bytebase will store data to the S3 bucket.")
+	rootCmd.PersistentFlags().StringVar(&flags.credentialsFile, "credentials-file", "", "credential file to use for the backup bucket. It should be the same format as the AWS credential files.")
```

Comment:
If it's backup specific, maybe we should just call it --backup-credential?

Reply:
Do you mean that if we later require a new cloud service x, we then add a new flag called `--x-credentials-file`?

Reply:
Nope, I just mean to use --backup-credential for all services.

credentials-file is a bit general as it doesn't convey `backup` and it's more likely we will use a different credential format when integrating with a different service.